_pydevd_bundle/pydevd_frame.py

```diff
@@ -255,6 +259,42 @@ def handle_exception(self, frame, event, arg):
             main_debugger = None
             thread = None
 
+    def manage_return_values(self, main_debugger, frame, event, arg):
+
+        def get_func_name(frame):
+            code_obj = frame.f_code
+            func_name = code_obj.co_name
+            try:
+                cls_name = get_clsname_for_code(code_obj, frame)
+                if cls_name is not None:
+                    return "%s.%s" % (cls_name, func_name)
+                else:
+                    return func_name
+            except:
+                traceback.print_exc()
+                return func_name
+
+        try:
+            if main_debugger.show_return_values:
+                if event == "return" and hasattr(frame, "f_code") and hasattr(frame.f_code, "co_name"):
+                    if hasattr(frame, "f_back") and hasattr(frame.f_back, "f_locals"):
+                        if RETURN_VALUES_DICT not in dict_keys(frame.f_back.f_locals):
+                            frame.f_back.f_locals[RETURN_VALUES_DICT] = {}
```

Comment:
Actually, better in this case would be getting with a default None value and if it returns None, set the initial RETURN_VALUES_DICT (i.e.: no need to check and later get).